container/root/etc/services-available/php-fpm/.command

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec php5-fpm -F -O 2>&1 | sed -u 's,.*: \"\(.*\)$,\1,'| sed -u 's,\"$,,'
```

Comment:
moved to a separate file to execute properly. did *not* work within pipeline or execlineb context